src/materials/nodes/Line2NodeMaterial.js

```diff
@@ -40,6 +40,7 @@ class Line2NodeMaterial extends NodeMaterial {
 		this.lineWidth = 1;
 
 		this.lineColorNode = null;
+		this.opacityNode = null;
```

Comment:
`opacityNode=null` seems unnecessary here, it is declared in `NodeMaterial` class.

Reply:
removed